src/ExecBlock/ExecBlockManager.cpp

```diff
@@ -216,50 +222,158 @@ size_t ExecBlockManager::searchRegion(rword address) const {
     LogDebug("ExecBlockManager::searchRegion", "Searching for address 0x%" PRIRWORD, address);
     // Binary search of the first region to look at
     while(low + 1 != high) {
-        size_t idx = (low + high) / 2;
-        if(regions[idx].covered.start > address) {
+        size_t idx = (low + high) >> 1;
```

Comment:
This optimization is automatically done by the compiler, you should keep ``/ 2`` to express you intent

Reply:
ack